### DISCUSSION TITLE
Recommendation Engine

### DISCUSSION THREAD

**Discussion by MarthinusBosman** [sct/overseerr#3562] [Feature Requests]
I'm really loving Overseerr so far, but the one thing it's missing for me, and is perfectly placed to do, is having a recommendation engine built in.

We already have users' plex watchlists, recent requests, and potentially their plex ratings. It would be great to just have a "recommended" section in Overseerr so I don't either need to know exactly what I want to watch, or just see the latest Movies/TV Shows which are going to hit every "trending/popular/upcoming" category.

I also wouldn't mind adding a PR myself if maintainers agree that this is something that fits within Overseerr?

**Comment by x86txt** [comment]
I bet this could even just be grabbed from the Radarr API for an initial effort, since Radarr has something similar built. At worst they could look at how Radarr implemented the discover engine for inspiration.
